test/connectors/mocks/MockCreateTrnService.scala

```diff
@@ -31,15 +31,20 @@ trait MockCreateTrnService extends MockitoSugar with BeforeAndAfterEach {
 
   val mockCreateTrnService: CreateTrnService = mock[CreateTrnService]
 
-  override protected def beforeEach(): Unit = {
-    super.beforeEach()
-    reset(mockCreateTrnService)
-  }
-
   def mockCreateTrn(journeyId: String)(response: Future[String]): OngoingStubbing[_] =
     when(
       mockCreateTrnService.createTrn(ArgumentMatchers.eq(journeyId)
       )(ArgumentMatchers.any[HeaderCarrier])
     ).thenReturn(response)
 
+  def verifyCreateTrn(journeyId: String): Unit =
```

Comment:
When I use verify I like to define the expected number of invocations of the method (i.e. verify(mockCreateTrnService, times(1))....